tests/integration_tests/performance/configs/test_block_performance_config_5.10.json

```diff
@@ -275,39 +275,39 @@
                                         "1vcpu_1024mb.json": {
                                             "Avg": {
                                                 "randrw-bs4096": {
-                                                    "target": 83319,
+                                                    "target": 58061,
```

Comment:
hm it's weird that the performance numbers reported here are worse with io_uring.
I'm fairly sure that this is due to the fact that we run those tests using a ramdisk.

We should gather those baselines on a nvme drive for io_uring.
If this is too much work for this PR, we may hardcode the use of the Sync backend in this test and work on it later

Reply:
NVMe breaks other things unfortunatelly.

Ideally there should be different profiles for each environment (i.e. ramdisk, nvme, etc.).